core/src/main/java/org/stellar/anchor/event/models/TransactionEvent.java

```diff
@@ -0,0 +1,141 @@
+package org.stellar.anchor.event.models;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.gson.annotations.SerializedName;
+import java.time.Instant;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class TransactionEvent implements AnchorEvent {
+  @SerializedName("event_id")
+  String eventId;
+
+  Type type;
+
+  public String getType() {
+    return this.type.type;
+  }
+
+  String id;
+
+  Sep sep;
```

Comment:
Can we use integer instead of the `Sep` class?

Reply:
i'll address this in a separate PR where i'll also update the transaction_status_changed schema